src/pkg/services/m365/api/lists.go

```diff
@@ -579,3 +579,15 @@ func ListToSPInfo(lst models.Listable) *details.SharePointInfo {
 		},
 	}
 }
+
+func listCollisionKeyProps() []string {
+	return idAnd("displayName")
+}
+
+func ListCollisionKey(list models.Listable) string {
+	if list == nil {
+		return ""
+	}
+
+	return ptr.Val(list.GetDisplayName())
```

Comment:
Is this a unique enough collision key? Wondering if we have converged on this as the collision key during design discussions.

Reply:
for lists,
- we cannot create 2 lists with same name --> hence going by the name itself as the collision key
- also we cannot set `name` attrbute of the list, we can only set `displayName`. `name ` is set based on display name.

Reply:
Thanks for clarifying! Could you please leave this note on top of this function?